packages/styleguide/src/theme/colors.js

```diff
@@ -203,6 +203,18 @@ const colors = {
     flyerMetaText: '#A3BBFF',
     flyerFormatText: '#D90034',
   },
+  climateColors: {
+    light: {
+      default: 'pink',
+      primary: 'blue',
+      primaryHover: 'yellow',
+    },
+    dark: {
+      default: 'purple',
+      primary: 'yellow',
+      primaryHover: 'darkblue',
+    },
+  },
```

Comment:
I am not really fond of having it in the default colors object. However I also don't see a better place, so I guess it's fine…
Perhaps use the identical color values as the default theme until the final KlimaLabor colors are defined?

Reply:
@ovbm What are your thoughts on this?